pkg/list/list_values.go

```
@@ -0,0 +1,424 @@
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+	"github.com/cloudposse/atmos/pkg/utils"
+)
+
+const (
+	DefaultCSVDelimiter = ","
+	DefaultTSVDelimiter = "\t"
+	TableColumnPadding  = 3
+)
+
+// getMapKeys returns a sorted slice of map keys
+func getMapKeys(m map[string]interface{}) []string {
```

golangci-lint (1.62.2)

func `getMapKeys` is unused

(unused)